packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -400,12 +400,24 @@
                     >
                         <oui-tile-description>
                             <div
-                                data-ng-if="hosting.sqlPriveInfo.nbDataBaseInclude > 0"
+                                data-ng-if="hosting.sqlPriveInfo.nbDataBaseActive > 0"
                                 data-wuc-progress-bar-element-counter
                                 data-wuc-pbec-nb-elements="hosting.sqlPriveInfo.nbDataBaseActive"
                                 data-wuc-pbec-max-elements="hosting.sqlPriveInfo.nbDataBaseInclude"
                                 data-wuc-pbec-upper-limit="cloudWebUnlimitedQuantity"
                             ></div>
+                            <p
+                                data-ng-if="hosting.sqlPriveInfo.nbDataBaseInclude === 0"
+                            >
+                                0
```

Comment:
Is there a reason this paragraph is not managed with the one below ?
No `0 base(s) de données SQL privée(s) non activée(s)` ?

Reply:
sometimes the offer don't have private sql to activate so this sentence sounds weird (it makes you think that you can have some sql database activated). Displaying a "0" is clearer